src/vis/vis.js

```diff
@@ -15,12 +15,19 @@ var LayersCollection = require('../geo/map/layers');
 var AnalysisPoller = require('../analysis/analysis-poller');
 var Layers = require('./vis/layers');
 
+var STATES = {
+  INIT: 'init', // vis hasn't been sent to Windshaft
+  OK: 'ok', // vis has been sent to Windshaft and everything is ok
+  ERROR: 'error' // vis has been sent to Windshaft and there were some issues
```

Comment:
Why not just use separate variables instead (e.g. `STATE_INIT`) which has the benefit that non-usages and typos are caught by the test suite (through the linter)? The only motivation I can see for using an object would be if the vars would be exposed outside the file's own scope.